execution/gethexec/sequencer.go

```diff
@@ -699,6 +707,7 @@ func (s *Sequencer) precheckNonces(queueItems []txQueueItem) []txQueueItem {
 	outputQueueItems := make([]txQueueItem, 0, len(queueItems))
 	var nextQueueItem *txQueueItem
 	var queueItemsIdx int
+	var totalBlockSize int
```

Comment:
I don't like kicking txes out of a block because a nonce hit.

What about this?
* Initialize totalBlockSize to size of all txes in original queueItems (can recompute or get as parameter) 
* when revivingFailure, check totalBlockSize + tevived item, and according to that either set nextQueueItem or send it to retryQueue